### PR TITLE
OCT-2162: Return winners details

### PR DESCRIPTION
**Pull request by kgarbacinski**
## Definition of Done

1. [ ] If required, the desciption of your change is added to the [QA changelog](https://www.notion.so/octantapp/Changelog-for-the-QA-d96fa3b411cf488bb1d8d9a598d88281) 
2. [ ] Acceptance criteria are met.
3. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
4. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
5. [ ] Unit tests are added unless there is a reason to omit them.
6. [ ] Automated tests are added when required.
7. [ ] The code is merged.
8. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
9. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: backend/app/infrastructure/routes/user.py [hunk 1]
```diff
@@ -316,3 +331,26 @@ def get(self, epoch: int):
                 for user_address, uq_score in uq_scores
             ]
         }
+
+
+@ns.route("/<string:user_address>/winnings/<int:epoch>")
+@ns.doc(
+    params={
+        "epoch": "Epoch number",
```
Comment (aziolek) [inline backend/app/infrastructure/routes/user.py:339]: Why epoch number? Raffles are not bound in any way to epochs and time winnings are locked for can be of any length.
Adding this `epoch` parameter here would require FE to iterate over all epochs to get all winnings.

Maybe there is something I don't see here.

File: backend/app/infrastructure/routes/user.py [hunk 2]
```diff
@@ -316,3 +331,26 @@ def get(self, epoch: int):
                 for user_address, uq_score in uq_scores
             ]
         }
+
+
+@ns.route("/<string:user_address>/winnings/<int:epoch>")
```
Comment (aziolek) [inline backend/app/infrastructure/routes/user.py:336]: I suggest adding "raffle" to the slug, e.g. `raffle/winnings`. "Winnings" are very broad and if we will introduce any other type of "winning" it'll get really confusing fast.

File: backend/app/infrastructure/routes/user.py (hunk 1 above)
Comment (kgarbacinski) [reply backend/app/infrastructure/routes/user.py]: Sure, changed.

File: backend/app/infrastructure/routes/user.py (hunk 2 above)
Comment (kgarbacinski) [reply backend/app/infrastructure/routes/user.py]: Changed.

**Review by adam-gf — APPROVED** [review]

**Review by adam-gf — APPROVED** [review]

**Pull request merged**